From f1c27fd4eaa59e86005f9698bb839451a2f0ef44 Mon Sep 17 00:00:00 2001
From: r-value <i@rvalue.moe>
Date: Wed, 17 Nov 2021 18:02:20 +0800
Subject: [PATCH] Remove incorrect arg for i18n.merge_file

`i18n.merge_file` has been ignoring positional arguments and
explicitly rejects with error "ERROR: Function does not take
positional arguments" since meson 0.60.0
---
 data/meson.build                 | 2 --
 plugins/gedit-plugin/meson.build | 1 -
 2 files changed, 3 deletions(-)

diff --git a/data/meson.build b/data/meson.build
index c6aeffb4..0bc531a2 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -19,7 +19,6 @@ appdata_conf = configuration_data()
 appdata_conf.set('application_id', APPLICATION_ID)
 appdata = APPLICATION_ID + '.appdata.xml'
 appdata_file = I18N.merge_file(
-  appdata,
   input: configure_file(
     input: 'org.gnome.Devhelp.appdata.xml.in.in',
     output: APPLICATION_ID + '.appdata.xml.in',
@@ -45,7 +44,6 @@ desktop_conf = configuration_data()
 desktop_conf.set('application_id', APPLICATION_ID)
 desktop = APPLICATION_ID + '.desktop'
 desktop_file = I18N.merge_file(
-  desktop,
   type: 'desktop',
   input: configure_file(
     input: 'org.gnome.Devhelp.desktop.in.in',
diff --git a/plugins/gedit-plugin/meson.build b/plugins/gedit-plugin/meson.build
index 4f779886..877a01ff 100644
--- a/plugins/gedit-plugin/meson.build
+++ b/plugins/gedit-plugin/meson.build
@@ -7,7 +7,6 @@ install_data(
 
 plugin_info_file = 'devhelp.plugin'
 I18N.merge_file(
-  plugin_info_file,
   type: 'desktop',
   input: plugin_info_file + '.desktop.in',
   output: plugin_info_file,
-- 
GitLab

